libs/pages/layout/src/lib/ui/layout-page/layout-page.tsx

```diff
@@ -53,6 +55,7 @@ export function LayoutPage(props: PropsWithChildren<LayoutPageProps>) {
   const { data: clusterStatuses } = useClusterStatuses({ organizationId, enabled: !!organizationId })
   const { data: organization } = useOrganization({ organizationId })
   const { roles, isQoveryAdminUser } = useUserRole()
+  const isFeatureFlag = useFeatureFlagVariantKey('devops-copilot')
```

Comment:
A more explicit name could be used here. But as we only have a single FF in the component, that's alright 👍